### ISSUE TITLE
Pulsar Support

### ISSUE THREAD

**Issue by kpullu** [zendesk/maxwell#1464]
Hey,
Currently we use Maxwell to push data to Kinesis, however we're planning to start using Apache Pulsar in the near future. Do you have support for Pulsar in the roadmap, please?

**Comment by osheroff** [comment]
Hi @kpullu, 
No, we have no official support planned for development.  Generally the recipe for getting a new producer into Maxwell has been:

1.  Somebody wants it.
2.  That same somebody writes an initial draft and is willing to use it in production.
3.  We accept a PR and then maintain the producer going forward.

So for a new producer to enter in, we very much need to have someone who's going to use it in production and work out all the bugs.  Development without production is... meh.

In other words, we'd love to see your PR.  Producers are generally pretty easy to write, and it's even possible to write them  just for yourself (using [custom producers](https://maxwells-daemon.io/producers/#custom-producer)) and then contribute it back to the community once it's mature.

Thanks for your interest,
osheroff

**Comment by kpullu** [comment]
Hi osheroff,

fully agree with your reasoning. Just wanted to confirm whether such producer exists before we start implementing our custom one and issue a PR. Will keep you updated with our progress.

Well done for the amazing Maxwell product
Thanks
